bpython/curtsiesfrontend/repl.py

```diff
@@ -775,6 +775,13 @@ def process_key_event(self, e):
             self.on_tab(back=True)
         elif e in key_dispatch[self.config.undo_key]:  # ctrl-r for undo
             self.prompt_undo()
+        elif e in ["<Ctrl-g>"]:  # for redo
```

Comment:
Also it'd be nice to
- follow the pattern of other keys of calling a method to keep this switch-like if/elif/elif/... structure shorter
- also for symmetry, use a tuple here instead of a list